### ISSUE TITLE
🐛 [Bug] URGENT; Last version of Remède landing on a blank page

### ISSUE THREAD

**Issue by camarm-dev** [camarm-dev/remede#220]
The latest version of Remède lands on a blank page, with no errors. Should investigate and resolve this critic issue quickly.

**Comment by camarm-dev** [comment]
Seems to be resolved ?

**Comment by Ilithy** [comment]
Hi, I have a similar problem, nothing works, not even requests for a word, whatever the type of request, likewise, downloading a dictionary returns an error.

**word search test**
![Screenshot_20241221-123534](https://github.com/user-attachments/assets/e05e8ad9-6f61-4378-96df-a88b31c91f79)

**"word of the day" test**
![Screenshot_20241221-122955](https://github.com/user-attachments/assets/49680feb-fa99-4438-9af9-f311ee9f6782)


**dictionary download test via app settings**
![Screenshot_20241221-122410](https://github.com/user-attachments/assets/faa0c969-74be-4c8e-ae24-dc2b8b94452a)

Thanks you

**Comment by camarm-dev** [comment]
Hi, this issue was resolved. The problem you are facing is related to what I explained in this comment https://github.com/camarm-dev/remede/issues/234#issuecomment-2558091933 . I updated the API during the night, because I'm about to release version 1.4.0-beta, but the new API does not works with old versions of the application. If you want, add me as a friend on discord (my pseudo is "camarm") and I will tell you more, and send you a build of the beta if you want to use Remède today.
